roles/keycloak/templates/15.0.2/standalone-ha.xml.j2

```diff
@@ -123,7 +124,7 @@
                 <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
             </formatter>
             <formatter name="COLOR-PATTERN">
-                <pattern-formatter pattern="%K{level}%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
+                <pattern-formatter pattern="%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
```

Comment:
It's better to leave the yyyy-MM-dd information in each log line (log files can be sent to remote syslog, concatenated processed etc)

Reply:
you 're right